bc_gym_planning_env/envs/base/env.py

```diff
@@ -129,13 +130,10 @@ def __init__(self, costmap, path, params, random_param=None):
         """
         # Stateful things
         self._robot = TricycleRobot(dimensions=get_dimensions_example(params.robot_name))
-        self._reward_provider = ContinuousRewardProvider(params=params.reward_provider_params)
+        self._reward_provider = ContinuousRewardPurePursuitProvider(params=params.reward_provider_params)
```

Comment:
Please do not change behaviour for everyone while introducing behaviour for your case. Please dependency inject RewardProvider, based on parametrization.

Reply:
fixed as suggested